README.md

```diff
@@ -279,16 +282,17 @@ The comprehensive reading list for Linux kernel-space programming. The choice is
 * [Linux Kernel Networking: Implementation and Theory ](https://www.amazon.com/dp/143026196X) (Rami Rosen, 2013)
 * [Linux Kernel Programming](https://www.amazon.com/dp/178995343X) (Kaiwan N Billimoria, 2021)
 * [Linux Kernel Programming Part 2 - Char Device Drivers and Kernel Synchronization](https://www.amazon.com/dp/180107951X) (Kaiwan N Billimoria, 2021)
+* [Linux Kernel Debugging](https://www.amazon.com/dp/1801075034) (Kaiwan N Billimoria, 2022)
 * [Mastering Linux Device Driver Development](https://www.amazon.com/dp/178934204X) (John Madieu, 2021)
-* [Linux Device Drivers Development: Develop customized drivers for embedded Linux](https://www.amazon.com/dp/1785280007) (John Madieu, 2017)
+* [Linux Device Drivers Development 2nd Edition](https://www.amazon.com/dp/1803240067) (John Madieu, 2022)
 * [Linux Device Driver Development Cookbook](https://www.amazon.com/dp/1838558802) (Rodolfo Giometti, 2019)
 
 ## C++ Embedded Programming
 
 C++ embedded programming, kernel-level with some restrictive specific.
 
-* [Mastering Embedded Linux Programming](https://www.amazon.com/dp/1784392537) (Chris Simmonds, 2015)
```

Comment:
Two consecutive books by Chris Simmonds were two deprecated editions of the same book, now both of them are replaced by the third edition.
This book is also the greatest embedded book I've ever read.